docs-v2/data/maturity.json

```diff
@@ -231,6 +231,11 @@
     "maturity": "alpha",
     "description": "User can generate a starter tekton pipeline using skaffold"
   },
+  "verify": {
+    "area": "skaffold verify",
```

Comment:
I think you can also add:
```
"dev": "x",
"debug": "x",
"run": "x",
"deploy": "x",
```

This will show up as:
![image](https://user-images.githubusercontent.com/39389231/183244531-0811d273-d041-4ee9-8f6b-8c474f9aa632.png)

Reply:
Oh cool, thanks!  Done